Automattic/vip-cli
# feat(dev-env): Do not prompt for missing values if stdin is not a TTY

## Description

When we run `vip dev-env create` in a non-interactive environment (e.g., CI), it dies when it needs to prompt for a non-specified parameter:

```
$ vip dev-env create < /dev/
Running validation steps...
✓ Check for docker installation
✓ Check for docker-compose installation
✓ Check access to docker for current user
✓ Check DNS resolution
This is a wizard to help you set up your local dev environment.

Sensible default values were pre-selected for convenience. You may also choose to create multiple environments with different settings using the --slug option.


? WordPress site tvolodymyr@democratic-circus:~/$
```

This PR adds a check whether `stdin` is a TTY and if not, disables interactive prompts.

## Steps to Test

1. Check out PR.
2. Run `npm run build`
3. Run `./dist/bin/vip-dev-env-create < /dev/null`
4. Verify the environment has been created and can be started.
